app/lib/actions/pulp/repository/create.rb

```diff
@@ -104,6 +105,14 @@ def iso_distributor
           end
         end
 
+        def puppet_distributor
+          options = { id: "#{input[:pulp_id]}_puppet", auto_publish: true }
+          Runcible::Models::PuppetDistributor.new(input[:path],
```

Comment:
I *think* input[:path] here is pointing to the puppet env in /etc/puppet/environments/SOMETHING  which may not make sense for the normal distributor.  It all seems to work, as it seems to be published in:

/var/lib/pulp/published/puppet/https/repos/Default_Organization-Library-asdfasdf/

properly, but i'm not fully sure what that option actually does.